docs/articles_en/documentation/openvino-ir-format/operation-sets/operation-specs/normalization/rms-norm-14.rst

```diff
@@ -0,0 +1,95 @@
+.. {#openvino_docs_ops_normalization_RMS_14}
+
+RMS
+===
+
+
+.. meta::
+  :description: Learn about RMS-14 - a normalization operation.
+
+**Versioned name**: *RMS-14*
+
+**Category**: *Normalization*
+
+**Short description**: Calculates Root Mean Square (RMS) normalization of the input tensor.
+
+**Detailed description**
+
+*RMSNorm* operation performs Root Mean Square (RMS) normalization on a given input ``data`` along dimensions specified by ``axes`` input.
+`Reference <https://arxiv.org/abs/1910.07467>`__.
+
+.. math::
+
+    (x / Sqrt(ReduceMean(x^2, axes) + eps))
+
+
+ -   If the optional ``scale`` input is provided:
+
+.. math::
+
+    (x / Sqrt(ReduceMean(x^2, axes) + eps)) * scale
```

Comment:
Is the final decision to have multiplication by `x` inside `RMSNorm`? Why?

Reply:
The discussion I mentioned in the PR description is about having the `scale` inside or outside the formula.
And I proposed to keep it optional for compatibility with existing GPU RMSNorm op.

Could you please precise, do you see other options for the RMSNorm formula?